src/lib/lightnet.js

```diff
@@ -17,10 +17,27 @@ const lightnetDockerContainerName = 'mina-local-lightnet';
 const lightnetMinaDaemonGraphQlEndpoint = 'http://localhost:8080/graphql';
 const lightnetAccountsManagerEndpoint = 'http://localhost:8181';
 const lightnetArchiveNodeApiEndpoint = 'http://localhost:8282';
+const lightnetProcessName = Constants.lightnetProcessName;
+const lightnetDockerProcessToLogFileMapping = new Map([
```

Comment:
This works, but I was curious if there was a specific reason you chose to use Maps here and in a few other places instead of objects?